src/pages/docs/supporting-projects/emulsify-core/advanced-usage/3rd-party-libraries.md

```diff
@@ -111,45 +115,23 @@ Add the new JS file to the `components/02-molecules/status/status.stories.js` an
 + };
 ```
 
-Now if you run the storybook `npm run develop` you should see a blinking error message in the status molecule.
-
-#### Add JS File to Theme Library and Load in the Template
-
-Add the JS file to the theme library by editing `MY_THEME.libraries.yml`:
+Now if you run the storybook `npm run develop` you should see a blinking error message in the status molecule. -->
 
-```text
-+
-+ status:
-+   js:
-+     dist/js/02-molecules/status/status.js: {}
-+   dependencies:
-+     - core/drupal
-+     - core/jquery
-```
-
-Make sure to include the drupal and jquery core libraries as dependencies. They will both be loaded before your JS file in Drupal.
-
-The library then needs to be loaded in the component twig template. In this case edit `components/02-molecules/status/status.stories.twig` and add:
```

Comment:
Adding this Drupal specific library attachment directly to a component twig file is not recommended. While this won't cause any errors it's mixing component files with CMS specific functions. Ideally, this is added to the template twig file this component is wired to.